Tests/RequestInterceptorTests.swift

```diff
@@ -563,6 +578,13 @@ final class InterceptorTests: BaseTestCase {
         XCTAssertEqual(result, RetryResult.doNotRetryWithError(RetryError()))
         XCTAssertTrue(result.error is RetryError)
         XCTAssertFalse(retrier2Called)
+        checkMemoryLeaksIfApplicable(interceptor)
+    }
+    
+    private func checkMemoryLeaksIfApplicable(_ object: AnyObject, file: StaticString = #file, line: UInt = #line) {
```

Comment:
Seems like this should just live in the `BaseTestCase` to allow for all tests to use it. And given we can't test back before iOS 13 anyway, I'm not sure the `IfApplicable` is really necessary here.

Reply:
Fixed